### ISSUE TITLE
Fences don't connect AOF-1.14.4-4.0.1

### ISSUE THREAD

**Issue by calloatti** [AK9Official/All-of-Fabric#25]
![2019-08-03_19 49 25](https://user-images.githubusercontent.com/12817862/62417518-1348dc80-b628-11e9-89ac-c9fb639ef2cf.png)

**Comment by calloatti** [comment]
![2019-08-03_19 52 15](https://user-images.githubusercontent.com/12817862/62417535-560ab480-b628-11e9-9c32-7ae7793713c3.png)

**Comment by Shnupbups** [comment]
Does this only affect vanilla wooden fences? Does this affect vanilla Nether Brick fences? Does this affect Extra Pieces fences? What about fences from other mods?

**Comment by coderbot16** [comment]
This is likely a Fabriblocks issue, as it overrides the fences tag without including the vanilla entries as fallback. Since wooden_fences is simply referenced by fences, the resource loader bug causes *only* Fabriblocks fences and fences under minecraft: fences directly to work.

**Comment by calloatti** [comment]
Disabled Fabriblocks and the issue persists:

![image](https://user-images.githubusercontent.com/12817862/62424857-0a4d1f00-b6ab-11e9-97e0-b50c2946bca1.png)

**Comment by coderbot16** [comment]
Try removing bambooeverything, it may override the tag as well.

**Comment by Shnupbups** [comment]
Why do i feel like it's gonna end up being my fault because i messed something up again?

**Comment by calloatti** [comment]
**Edit: This test is flawed, new fences have to be placed to test**

fabriblocks disabled
bambooeverything disabled

![2019-08-04_11 37 59](https://user-images.githubusercontent.com/12817862/62424978-5e0c3800-b6ac-11e9-90c1-336d442ac0f7.png)

*** 

fabriblocks disabled
bambooeverything disabled
extrapieces disabled (not your fault extra pieces!)
![2019-08-04_11 40 52](https://user-images.githubusercontent.com/12817862/62425021-ba6f5780-b6ac-11e9-8ffb-a909aaf62fb9.png)

Any more ideas? What strings should I search for in the mod jars to find mods that may be messing this up?

**Comment by coderbot16** [comment]
If a mod jar contains a file under data/minecraft/tags/blocks/fences.json, it could cause the issue. You could try removing extra pieces but it probably doesn't cause the issue.

As always, the root issue is a bad jij of resource loader.

**Comment by Shnupbups** [comment]
Yeah, any mod that contains a `data\minecraft\tags\blocks\fences.json` is a potential culprit.
Glad it's not EP though!

**Comment by coderbot16** [comment]
I'm not saying it's 100% not EP, just that it's unlikely... You never know what odd stuff can go on.

**Comment by Shnupbups** [comment]
> I'm not saying it's 100% not EP, just that it's unlikely... You never know what odd stuff can go on.

Callo removed EP in the post above yours and confirmed it isn't the problem

**Comment by coderbot16** [comment]
ah nvm

**Comment by calloatti** [comment]
Well, all my testing was flawed, you have to place new fences to check if the problem was fixed.

Found this in fabriblocks fences.json

```
{
	"replace": false,
	"values": [
		"fabriblocks:red_nether_brick_fence"
	]
}
```

And this in bambooeverything fences.json

```
{
  "replace": false,
  "values": [
    "bambooeverything:bamboo_fence"
  ]
}
```
Those are the only 2 mods in the pack that have a fences.json

Disabling both mods makes fences connect again

Enabling only bambooeverything breaks fences (fabriblocks disabled)

Enabling only fabriblocks breaks fences (bambooeverything disabled)

So bottom line, both mods have to be disabled to fix the problem

Image showing both working and broken fences, just to show you have to place new fences to test:

![2019-08-04_12 15 59](https://user-images.githubusercontent.com/12817862/62425601-a417ca80-b6b1-11e9-83b7-d95a26bd229c.png)

**Comment by calloatti** [comment]
Temporary fix in https://github.com/AllOfFabric/AOF/commit/5babdf4fa33328880b1508063a821815039dc4ad
